Doc/c-api/intro.rst

```diff
@@ -174,6 +174,23 @@ complete listing.
    .. versionchanged:: 3.8
       MSVC support was added.
 
+.. c:macro:: PyDoc_STRVAR(name, str)
+
+   Creates a variable with name ``name`` that can be used in docstrings.
```

Comment:
Formatting nit, argument names should be *italicized* instead of in ``codeblocks``. Also, I think "called name" reads a bit better than "with name name".
```suggestion
   Creates a variable called *name* that can be used in docstrings.
```